src/hotspot/share/logging/logAsyncWriter.cpp

```diff
@@ -149,6 +150,9 @@ void AsyncLogWriter::write() {
 
   if (req > 0) {
     assert(req == 1, "AsyncLogWriter::flush() is NOT MT-safe!");
+    // LogFileOutput::write_block() has called fflush().
+    // stderr does not cache.
```

Comment:
`buffer` is the right word.LogFileOut::write_block() has called fflush() to flush all file-based streams.
LogStdoutOutput/LogFileStreamOutput haven't because of `LogFileStreamOutput::write_blocking()`. 

stderr is unbuffered, so I think we only need to flush stdout here.